### PR TITLE
Update readme.md, explaining how to obtain the path/to/control.unit.sock and also fixed wrong ports on samples

### PR DESCRIPTION
**Pull request by 0x6f677548**

### CODE REVIEW

**Review by ac000 — CHANGES_REQUESTED** [review]
Hi,

Please undo this change in the previous commit and then just drop this one.

**Review by ac000 — CHANGES_REQUESTED** [review]
Seeing as '8080' is used in a couple of places. I'd just change the '8000' in the PUT command

```shell
# curl -X PUT -d '{"127.0.0.1:8000": {"pass": "applications/helloworld"}}'  \
       --unix-socket /path/to/control.unit.sock http://localhost/config/listeners
```


to '8080'.

**Comment by 0x6f677548** [comment]
> Seeing as '8080' is used in a couple of places. I'd just change the '8000' in the PUT command
> 
> ```shell
> # curl -X PUT -d '{"127.0.0.1:8000": {"pass": "applications/helloworld"}}'  \
>        --unix-socket /path/to/control.unit.sock http://localhost/config/listeners
> ```
> 
> to '8080'.

either 8000 or 8080 should be ok if used consistently through the samples. My point was that both ports were being used on the same guide (readme.md) which could be confusing for some newcomers.

**Comment by ac000** [comment]
> either 8000 or 8080 should be ok if used consistently through the samples. My point was that both ports were being used on the same guide (readme.md) which could be confusing for some newcomers.

Yes, exactly!

So rather than change the 8080's to 8000, change the single 8000 to 8080...

File: README.md [hunk 1]
```diff
@@ -123,7 +133,7 @@ Now Unit accepts requests at the specified IP and port, passing them to the
 application process. Your app works!
 
 ``` console
-$ curl 127.0.0.1:8080
+$ curl 127.0.0.1:8000
 
       Hello, PHP on Unit!
 ```
```
Comment (ac000) [inline README.md:139]: Drop this hunk.

**Review by ac000 — CHANGES_REQUESTED** [review]

**Review by ac000 — CHANGES_REQUESTED** [review]
Change the last commit to change the port to 8080 in the curl PUT command.

**Comment by 0x6f677548** [comment]
I accidentally deleted patch branch while rebasing. Re-submitted new pull request #816

**Pull request closed without merging**